deploy/runtime/defaults/PV Battery_Commercial.txt

```diff
@@ -1,5 +1,5 @@
 1
-1058
+1073
```

Comment:
Defaults note: PV Battery Commercial has manual schedule with 4 weekday periods and only 2 periods enabled. Manual dispatch is not the default dispatch mode for PV Battery Commercial

Reply:
Peak shaving as the default dispatch is intentional.
Great point about only having two periods of discharging, I'll get that updated such that it always discharges during the high cost period.